README.md

```diff
@@ -19,10 +19,49 @@ Using GraphQL with server-side rendering in React is a challenging problem. Curr
 
 ### Installation
 
+Install `next-urql` along with its `peerDependencies`.
+
 ```sh
-yarn add next-urql
+yarn add next-urql react-is styled-components isomorphic-unfetch
+```
+
+`react-is`, `styled-components`, and `isomorphic-unfetch` help to support server-side `Suspense` with `react-ssr-prepass`. This assumes you have followed the basic installation steps for `urql` [here](https://github.com/FormidableLabs/urql#installation).
+
+### Usage
+
+To use `next-urql`, first `import` the `withUrqlClient` higher order component.
+
+```javascript
+import { withUrqlClient } from 'next-urql';
+```
+
+Then, for any page in your `pages` directory for which you want to prefetch GraphQL queries, wrap the page in `withUrqlClient`. For example, let's say you have an `index.js` page that renders two components that make GraphQL requests using `urql`, `PokemonList` and `PokemonTypes`. To run their queries initially on the server-side you'd do something like the following:
```

Comment:
Maybe we should add a disclaimer that if they wrap `_app` they will opt-out of static page optimizations due to this injecting `getInitialProps` on every page

Reply:
Will do! Good call out. The Caveats section hints at this but it'd be good to make it more explicit.